develop/toolkit/blockchain/spawn-testing-chain.md

```diff
@@ -1,4 +1,1213 @@
+??? interface "`-m`, `--monitor` - start as monitor, don't auto clean up network"
+
+    === "Argument"
+
+        None
+
+??? interface "`-c`, `--spawn-concurrency` - number of concurrent spawning processes to launch. Defaults to `1`"
+
+    === "Argument"
+
+        - `<concurrency>` - desired quantity of processes
+
+??? interface "`-h`, `--help` - display help for command"
+
+    === "Argument"
+
+        None
+
+## Configuration Files
+
+The network configuration can be given in either JSON or TOML format. The Zombienet repository also provides a collection of [example configuration files](https://github.com/paritytech/zombienet/tree/main/examples){target=_blank} that can be used as a reference.
+
+!!! note
+    Each section may include provider-specific keys that aren't recognized by other providers. For example, if you use the local provider, any references to images for nodes will be disregarded.
+
+### Settings
+
+Through the keyword `settings`, it's possible to define the general settings for the network. The available keys are:
+
+- `global_volumes?` ++"GlobalVolume[]"++ - a list of global volumes to use 
+
+    ??? interface "`GlobalVolume` interface definition"
+        ```js
+        export interface GlobalVolume {
+        name: string;
+        fs_type: string;
+        mount_path: string;
+        }
+        ```
+
+- `bootnode` ++"boolean"++ - add bootnode to network. Default is `true`
+- `bootnode_domain?` ++"string"++ - domain to use for bootnode
+- `timeout` ++"number"++ - global timeout to use for spawning the whole network"
+- `node_spawn_timeout?` ++"number"++ - timeout to spawn pod/process
+- `grafana?` ++"boolean"++ - deploy an instance of Grafana
+- `prometheus?` ++"boolean"++ - deploy an instance of Prometheus
+- `telemetry?` ++"boolean"++ - enable telemetry for the network
+- `jaeger_agent?` ++"string"++ - the Jaeger agent endpoint passed to the nodes. Only available on Kubernetes
+- `tracing_collator_url?` ++"string"++ - the URL of the tracing collator used to query by the tracing assertion. Should be tempo query compatible
+- `tracing_collator_service_name?` ++"string"++ - service name for tempo query frontend. Only available on Kubernetes. Defaults to `tempo-tempo-distributed-query-frontend`
+- `tracing_collator_service_namespace?` ++"string"++ - namespace where tempo is running. Only available on Kubernetes. Defaults to `tempo`
+- `tracing_collator_service_port?` ++"number"++ - port of the query instance of tempo. Only available on Kubernetes. Defaults to `3100`
+- `enable_tracing?` ++"boolean"++ - enable the tracing system. Only available on Kubernetes. Defaults to `true`
+- `provider` ++"string"++ - provider to use. Default is `kubernetes`"
+- `polkadot_introspector?` ++"boolean"++ - deploy an instance of polkadot-introspector. Only available on Podman and Kubernetes. Defaults to `false`
+- `backchannel?` ++"boolean"++ - deploy an instance of backchannel server. Only available on Kubernetes. Defaults to `false`
```

Comment:
I'll let @eshaben weigh in on this ... Do we want to include the ? to show optionality or better to explicitly write out, `Optional`

Reply:
Yeah, explicitly write it out:

```
- `backchannel` ++"boolean"++ - *(optional)* deploy an instance of backchannel server. Only available on Kubernetes. Defaults to `false`
```